### PR TITLE
chore: provide resource pool id and namespace in env create

### PR DESCRIPTION
**Pull request by cmoog**
We can expose these values via a flag later, but for now using the default pool is OK.

### CODE REVIEW

File: internal/coderutil/env.go [hunk 1]
```diff
@@ -58,3 +58,17 @@ func EnvsWithProvider(ctx context.Context, client coder.Client, envs []coder.Env
...
+	if err != nil {
+		return nil, err
+	}
+	for _, p := range provider {
+		if p.Local {
+			return &p, nil
+		}
+	}
+	return nil, coder.ErrNotFound
+}
```
Comment (cmoog) [inline internal/coderutil/env.go:74]: cc @f0ssel

**Review by f0ssel — APPROVED** [review]
looks good for now, when we'll do the local -> default migration we can also consider whether we can rip out the `Local` bool altogether as well, but that's down the road.

**Review by Emyrk — COMMENTED** [review]
I know it's already merged, but :+1:

**Comment by Emyrk** [comment]
I know it's already merged, but :+1:

**Pull request merged**